Sources/StreamChat/APIClient/Endpoints/Payloads/MessageAttachmentPayload.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 
 /// A type that describes attachment JSON payload.
-struct AttachmentPayload {
+struct MessageAttachmentPayload {
```

Comment:
I'm only not sure about this naming, it feels odd to see the `Message` without the `Chat` prefix, since everywhere in the codebase we use `ChatMessage`. 🤔  Although I know `Message` is not related to the model per se in this case, this will confuse us and the customers. From what I understand, this is like the "Default" attachment payload, so why not keep it simple and name it `DefaultAttachmentPayload`?  WDYT?